2-dom-manipulation/form.js

```diff
@@ -17,37 +16,51 @@ function validateForm(event) {
         courses: document.querySelectorAll('input[type="checkbox"]'),
         message: document.querySelector('#texta')
     };
-    printInHTML(form.name, form.email, form.phone, form.courses, form.message);
+    deleteAllInformationParagraphs();
+    let name = validateName(form.name);
+    let email = validateEmail(form.email);
+    let phone = validatePhoneNumber(form.phone);
+    let courses = Array.isArray(validateCourses(form.courses));
+    if (name && email && phone && courses) {
+        printInHTML(form.name.value);
+        printInHTML(form.email.value);
+        printInHTML(form.phone.value);
+        printInHTML(validateCourses(form.courses));
+        printInHTML(form.message.value);
+    }
 }
 
-function validateName(nameToValidate) {
-    if (nameToValidate.value == '') {
-        nameToValidate.classList.add('name-error');
-        return 'Please fill the name field !';
+function validateName(name) {
+    if (name.value == '') {
+        name.classList.add('name-error');
+        printInHTML('Please fill the name field !');
     }
-    else if (!NAME_V.test(nameToValidate.value)) {
-        nameToValidate.classList.add('name-error');
-        return 'Please write a valid name, not numbers or special characters !';
+    else if (!NAME_V.test(name.value)) {
+        name.classList.add('name-error');
+        printInHTML('Please write a valid name, not numbers or special characters !');
     }
-    nameToValidate.classList.remove('name-error');
-    return nameToValidate.value;
+    name.classList.remove('name-error');
```

Comment:
Here you are removing the class even if it's bad.

Reply:
How can i specify to change the styles, "remove the error red border" ?